games-puzzle/icebreaker/icebreaker-1.9.7.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A classic puzzle game"
+DESCRIPTION="So, uh, there's a bunch of penguins on an iceberg in \
+Antarctica. You have been selected to catch them so they can be shipped \
+to Finland, where they are essential to a secret plot for world domination."
+HOMEPAGE="https://mattdm.org/icebreaker/"
+COPYRIGHT="2000-2002 Matthew Miller"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://mattdm.org/icebreaker/1.9.x/icebreaker-1.9.7.tgz"
+CHECKSUM_SHA256="ca8b344a6a37b2041fba83271ae5e7566cbc8dc4c3367addd46f2a2f3573017e"
+GLOBAL_WRITABLE_FILES="settings/icebreaker.scores keep-old"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+
+PROVIDES="
+	icebreaker = $portVersion
+	app:IceBreaker = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libsdl_mixer
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	devel:libsdl_mixer
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make prefix=$prefix bindir=$appsDir datadir=$dataDir \
+		highscoredir=$settingsDir mandir=$manDir
+}
+
+INSTALL()
+{
+	make prefix=$prefix bindir=$appsDir datadir=$dataDir \
+		highscoredir=$settingsDir mandir=$manDir install
+	mv $appsDir/icebreaker $appsDir/IceBreaker
+
+	addAppDeskbarSymlink $appsDir/IceBreaker IceBreaker
```

Comment:
Would be nice if there was a mutual understanding in this, for the moment ScummVM also resides as "ScummVM" in the $appsDir folder (instead of "scummvm") ...